### PR TITLE
Guernsey: Add new term

### PR DESCRIPTION
**Pull request by ondenman**
Changes scraper to pull in new term data.

Closes: #18489

### CODE REVIEW

**Comment by ondenman** [comment]
@tmtmtmtm The legislature consists of 38 members + 2 Alderney representatives. (http://www.bbc.co.uk/news/world-europe-guernsey-33421756) There was a by-election on the 19th October 2016. We've failed to capture the outgoing member as he is no longer listed on the site.

**Comment by ondenman** [comment]
...adding the deceased member to a manual file

File: data/Guernsey/States/sources/manual/members-2016.csv [hunk 1]
```diff
@@ -0,0 +1,2 @@
+id,area,party,email,twitter,linkedin,term,source,name
+"jones,_david_b.",Vale,Independent,David.Jones@deputies.gov.gg,https://twitter.com/DeputyDaveJones,"",2016,,David B. Jones
```
Comment (tmtmtmtm) [inline data/Guernsey/States/sources/manual/members-2016.csv]: Would be good to add an `end_date` to this if you can find it

**Comment by ondenman** [comment]
Closing in favour of https://github.com/everypolitician/everypolitician-data/pull/19089#issuecomment-255692824

**Pull request closed without merging**